### PR TITLE
feat: add slides and images

### PR DESCRIPTION
**Pull request by captcodemonkey**
Add copy and images for slides 5-8
22-25 Problem statement
Template links on references slide

### CODE REVIEW

File: assets/nvmrc.png
Comment (RyanParsley) [inline assets/nvmrc.png:1]: The redacted bit draws my eye a bit. I feel like you want devs to look at the node version. Maybe an underline there?

File: slides.md [hunk 1]
```diff
@@ -96,18 +105,40 @@ Write
...
-Premature optimization is bad. Bad in app architecture and bad in the broader persuit of productivity. Why? Because you are good at a lot of things, but predicting the future isn't one of them.
+Premature optimization is bad. Bad in app architecture and bad in the broader persuit of productivity. 
+
+Why? Because you are good at a lot of things, but predicting the future isn't one of them.
 
 ---
 
 background-image: url(./assets/evolutionOfAutomation.jpg)
-background-size: 450px auto
+background-size: auto 450px
```
Comment (RyanParsley) [inline slides.md]: did you double check that inverting these does the right thing when your resize the window? I tend to just adjust width and let the height sort itself out.

File: slides.md [hunk 2]
```diff
@@ -96,18 +105,40 @@ Write
...
 
 # Evolution of Productivity
 
 ???
 
-It's sort of an evolution. It's just that you don't want to race to the right side of this like may be implied.
+Natural selection is considered the main mechanism that causes evolution.
+
+Delighted-ness & Productivity should drive the evolution of your tools.
```
Comment (RyanParsley) [inline slides.md]: I think you want delightfullness or maybe just delight. This reads like you're saying your tools are delighted.

File: slides.md [hunk 3]
```diff
@@ -96,18 +105,40 @@ Write
...
+
+You can write a script because it delights you even if it doesn't make you more productive now.
```
Comment (RyanParsley) [inline slides.md]: maybe something like 

Replacing mindless monotony with a delightful script is a win even if the script doesn't save you a tone of time. You're more engaged with the task at hand, you probably understand it better. Just don't go overboard and remember that all of this is in service of shipping code people actually asked for.

File: slides.md [hunk 4]
```diff
@@ -298,8 +329,54 @@ The return on investment for such a change is exponential.
...
 ---
 
+# Problem
+
+## inconsistent node versions
+
+???
+Not using the same node version at all points causes issues.
+
+Locally, in pipelines, and release stages.
```
Comment (RyanParsley) [inline slides.md]: Why should I care? It's usually fine. /s

File: slides.md [hunk 5]
```diff
@@ -71,19 +71,28 @@ As such this presentation is less about trying to prescribe tools to you and mor
...
 - snippets in your editor and beyond
+- templates
 - shell aliases
 - shell scripts
 - language scripts (rust, python)
 - dotfiles
 
 ???
 
-When I talk about automation, I'm not talking Unit tests or home kit. I mean anytime you save a few keystrokes by using technology.
```
Comment (RyanParsley) [inline slides.md:81]: I like this, you don't need to capitalize Unit, but also this is speaker notes no one sees but us.

File: assets/nvmrc.png
Comment (captcodemonkey) [reply assets/nvmrc.png:1]: Fixed.

File: slides.md (hunk 2 above)
Comment (captcodemonkey) [reply slides.md]: I was going for a funny wording but it didn't land, updated.

File: slides.md (hunk 4 above)
Comment (captcodemonkey) [reply slides.md]: Added some different text.

**Pull request merged**